sceptre/config.py

```diff
@@ -231,7 +231,7 @@ def resolver_constructor_factory(node_class):
         self.add_yaml_constructors(
             resolvers_folder, Resolver, resolver_constructor_factory
         )
-        external_resolver_folder = os.path.join(os.getcwd(), "resolvers")
+        external_resolver_folder = os.path.join(str(self.sceptre_dir), "resolvers")
```

Comment:
I had to use `str(...)` to pass the check in `get_subclasses`, which expects a `str` argument. Let me know if there's a better way of dealing with this.